channeldb/invoices.go

```diff
@@ -1737,3 +1737,112 @@ func setSettleMetaFields(settleIndex kvdb.RwBucket, invoiceNum []byte,
 
 	return nil
 }
+
+// InvoiceDeleteRef holds a refererence to an invoice to be deleted.
+type InvoiceDeleteRef struct {
+	// PayHash is the payment hash of the target invoice. All invoices are
+	// currently indexed by payment hash.
+	PayHash lntypes.Hash
+
+	// PayAddr is the payment addr of the target invoice. Newer invoices
+	// (0.11 and up) are indexed by payment address in addition to payment
+	// hash, but pre 0.8 invoices do not have one at all.
+	PayAddr *[32]byte
+
+	// addIndex is the add index of the invoice.
```

Comment:
`godoc` comment and the variable name are mismatched.

Reply:
done